src/api-tmdb/api.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
```

Comment:
name of folder is not so good it should be api/index.ts where api is a folder and index is a file in that